src/Tizen.Applications.EventManager/Tizen.Applications/SystemEvents.cs

```diff
@@ -0,0 +1,905 @@
+﻿/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+namespace Tizen.Applications.EventManager
+{
+    /// <summary>
+    /// Supported System Events
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public static class SystemEvents
+    {
+        /// <summary>
+        /// Class for system-event of battery charger status
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public static class BatteryChargerStatus
```

Comment:
We do not allow nested class

Reply:
Then, could you suggest me the appropriate class?